cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -1,246 +0,0 @@
-/*
- * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
- */
-
-package com.linkedin.kafka.cruisecontrol.servlet;
-
-import com.codahale.metrics.Meter;
-import com.codahale.metrics.MetricRegistry;
-import com.codahale.metrics.Timer;
-import com.linkedin.cruisecontrol.common.config.ConfigException;
-import com.linkedin.cruisecontrol.servlet.EndPoint;
-import com.linkedin.cruisecontrol.servlet.handler.Request;
-import com.linkedin.cruisecontrol.servlet.parameters.CruiseControlParameters;
-import com.linkedin.kafka.cruisecontrol.async.AsyncKafkaCruiseControl;
-import com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig;
-import com.linkedin.kafka.cruisecontrol.config.RequestParameterWrapper;
-import com.linkedin.kafka.cruisecontrol.config.constants.WebServerConfig;
-import com.linkedin.kafka.cruisecontrol.servlet.purgatory.Purgatory;
-import java.io.IOException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.ExecutionException;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import static com.linkedin.kafka.cruisecontrol.KafkaCruiseControlUtils.KAFKA_CRUISE_CONTROL_SERVLET_SENSOR;
-import static com.linkedin.kafka.cruisecontrol.servlet.CruiseControlEndPoint.REVIEW;
-import static com.linkedin.kafka.cruisecontrol.servlet.CruiseControlEndPoint.REVIEW_BOARD;
-import static com.linkedin.kafka.cruisecontrol.servlet.KafkaCruiseControlServletUtils.*;
-import static com.linkedin.kafka.cruisecontrol.servlet.parameters.ParameterUtils.hasValidParameterNames;
-
-
-/**
- * The servlet for Kafka Cruise Control.
- */
-public class KafkaCruiseControlServlet extends HttpServlet {
```

Comment:
Where did you move this class to? E.g. since the change is backwards compatible, the servlet should still be kept somewhere.

Reply:
Alright, looks like it's moved to ServletRequestHandler

Reply:
Yes, that's right.